rule-packs/mitre-attck-lateral-movement-attack-paths.json

```diff
@@ -0,0 +1,590 @@
+[
+      {
+        "name": "lateral-movement-exploitation-of-remote-services-minimize-service-account-permissions",
+        "description": "M1048 - Application Isolation and Sandboxing - Minimize permissions and access for service accounts to limit impact of exploitation.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND google_iam_service_account AS acct THAT ASSIGNED AS rel google_iam_binding AS binding RETURN acct.displayName, count(rel)",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "LOW"
```

Comment:
INFO for these types of aggregation queries, they're not really actionable from a triage workflow perspective

Reply:
changed to info

Reply:
corrected